includes/llms.functions.core.php

```diff
@@ -115,47 +115,24 @@ function llms_content( $content = '' ) {
 /**
  * Provide deprecation warnings
  *
- * Very similar to https://developer.wordpress.org/reference/functions/_deprecated_function/
+ * This function uses WP core's `_deprecated_function()`, logging to the LifterLMS log file
+ * located at `wp-content/updloads/llms-logs/llms-{$hash}.log` instead of `wp-content/debug.log`.
```

Comment:
I was wondering if we could take this as an occasion to stop logging deprecation warnings in llms-log.
As developer I wouldn't want to go look into any plugin log file for that kind of stuff. What do you think?
But anyways, for the purpose of this PR it's fine so.

Reply:
Good point, I think you're right and realistically I was thinking we shouldn't change because of a "bacwards compat" issue but it's actually not a breaking change unless you consider "breaking" to mean "I have to look where I'd expect to look as a new dev and now it's there instead of somewhere else"

It's possible that a veteran might be slightly confused but a veteran is likely looking at debug.log too so there's nothing actually broken.

Updated and pushed.